org.eclipse.xtext.ui.testing/src/org/eclipse/xtext/ui/testing/AbstractQuickfixTest.xtend

```diff
@@ -116,7 +116,7 @@ abstract class AbstractQuickfixTest extends AbstractEditorTest {
 		val originalText = document.get
 		val issue = document.getValidationIssue(issueCode)
 
-		val actualIssueResolutions = issue.getResolutions
+		val actualIssueResolutions = issue.getResolutions.sortBy[label]
```

Comment:
I would sort not only the actual, but also the expected quickfixes.

Reply:
Well there is no additional value in that. Thus I keep it one side

Reply:
You can come up with a second or after this one

Reply:
If you sort the expected values here, there is no need to modify the concrete test cases.

Reply:
Can you please come up with ape and we will close this one